website/content/plugins/author/device.mdx

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+page_title: Create a device plugin for Nomad
+description: |-
+  Learn how to create a Nomad device plugin so you can schedule workload tasks with other devices, such as GPUs. Review device plugin lifecycle and the device plugin API functions that you must implement in your device plugin.
+---
+
+# Create a device plugin
+
+This page provides conceptual information for creating a device driver plugin to extend Nomad's workload execution functionality.
+
+Nomad has built-in support for scheduling compute resources such as CPU, memory,
+and networking. Use Nomad device driver plugins to support scheduling tasks with
+other devices, such as GPUs. Device driver plugins are responsible for
+fingerprinting these devices and working with the Nomad client to make them
+available to assigned tasks.
+
+For a real world example of a Nomad device plugin implementation, refer to the
+[Nvidia GPU plugin](https://github.com/hashicorp/nomad-device-nvidia).
+
+## Authoring a device plugin
+
+Authoring a device plugin in Nomad consists of implementing the
+[DevicePlugin][deviceplugin] interface alongside a main package to launch the
+plugin.
+
+The [nomad-skeleton-device-plugin][skeletonproject] exists to help bootstrap
+the development of new device plugins. It provides most of the boilerplate
+necessary for a device plugin, along with detailed comments.
+
+### Lifecycle and state
+
+A device plugin is long-lived. Nomad ensures that one instance of the plugin
+is running. If the plugin crashes or otherwise terminates, Nomad launches
+another instance of it.
+
+However, unlike [task driver plugins](/nomad/plugins/author/task-driver), device
+plugins do not currently have an interface for persisting state to the Nomad
+client. Instead, the device plugin API emphasizes fingerprinting devices and
+reporting their status. After helping to provision a task with a scheduled
+device, a device plugin does not have any responsibility, or ability, to monitor
+the task.
+
+## Device driver plugin API
+
+@include 'plugins/base.mdx'
```

Comment:
Similar for my comment about this inline in the driver plugin.

Reply:
done